content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -659,6 +668,45 @@ there is no automatic Job restart once the Job status is `type: Failed`.
 That is, the Job termination mechanisms activated with `.spec.activeDeadlineSeconds`
 and `.spec.backoffLimit` result in a permanent Job failure that requires manual intervention to resolve.
 
+### Terminal Job conditions
+
+A Job has two possible terminal states, it ends up either succeeded, or failed,
+and these states are reflected by the presence of the Job conditions `Complete`
+or `Failed`, respectively.
+
+The failure scenarios encompass:
+- the `.spec.backoffLimit`
+- the `.spec.activeDeadlineSeconds` is exceeded
+- the `.spec.backoffLimitPerIndex` is exceeded (see [Backoff limit per index](#backoff-limit-per-index))
+- the Pod matches the Job Pod Failure Policy rule with the `FailJob` action  (see more [Pod failure policy](#pod-failure-policy))
+
+The success scenarios encompass:
+- the `.spec.completions` is reached
+- the criteria specified by the Job Success Policy are met (see more [Success policy](#success-policy))
+
+### Termination of Job pods
+
+Prior to v1.31 the Job terminal conditions are added when the Job termination
+process is triggered, and all Pod finalizers are removed, but some pods may
+still remain running at that point in time.
+
+Since v1.31, when you enable either the `JobManagedBy` or
+`JobPodReplacementPolicy` (enabled by default)
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/), the
+Job controller awaits for termination of all pods before adding a condition
+indicating that the Job is finished (either `Complete` or `Failed`).
+
+Note that, the process of terminating all pods may take a substantial amount
```

Comment:
Additionally, using the Hugo short code would be better, right?
But I'm not sure if we should use the Hugo shortcode in all places where we want to mention the Note.

Reply:
Actually, the paragraph seems pretty important for the section so tentatively I remove the "Note that", and don't add the Hugo either. PTAL.

Reply:
SGTM